test/e2e-common.sh

```diff
@@ -33,16 +33,49 @@ readonly EVENTING_CONFIG="config/"
 # In-memory channel CRD config.
 readonly IN_MEMORY_CHANNEL_CRD_CONFIG_DIR="config/channels/in-memory-channel"
 
-# Setup the Knative environment for running tests.
-function knative_setup() {
-  # Install the latest Knative/eventing in the current cluster.
-  echo ">> Starting Knative Eventing"
-  echo "Installing Knative Eventing"
-  ko apply -f ${EVENTING_CONFIG} || return 1
+# Latest release. If user does not supply this as a flag, the latest
+# tagged release on the current branch will be used.
+readonly LATEST_RELEASE_VERSION=$(git describe --match "v[0-9]*" --abbrev=0)
```

Comment:
Is this the latest release version for only eventing? or are we using this for serving as well? What happens when serving or eventing has mismatching minor releases?

For example, latest current serving release is `v0.11.1` while eventing is still `v0.11.0`

Reply:
This is original code. I'm just installing the same version of serving. I think it would work as we are not testing serving per se here.

We of course might resolve the latest serving release independently. Not sure is it worth it.

Reply:
This is resolved by 3e03bc1

Reply:
Serving was removed from installation, so this relates to just eventing, now.